src/QA.Contribution.Test.Journey/Domain/ContactForm.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace QA.Contribution.Test.Journey.Domain
+{
+    internal class ContactForm
```

Comment:
created domain class to transform the values sent from feature file to step definition.